services/src/main/java/org/keycloak/crypto/ServerAsymmetricSignatureVerifierContext.java

```diff
@@ -21,6 +21,10 @@
 
```

Comment:
How about using `ClientAsymmetricSignatureVerifierContext` that is used for keycloak to verify the data that a client creates and signs (e.g. request object, JWS client assertion for client authentication)?

`ServerAsymmetricSignatureVerifierContext` is used for keycloak to sign and verify JWS (e.g. access token, ID token). DPoP Proof is the data that a client creates and signs and keycloak verify.